aws/table_aws_inspector2_member.go

```diff
@@ -0,0 +1,119 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/inspector2"
+
+	inspector2v1 "github.com/aws/aws-sdk-go/service/inspector2"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsInspector2Member(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_inspector2_member",
+		Description: "AWS Inspector2 Member",
+		List: &plugin.ListConfig{
```

Comment:
@ParthaI I think we should add support for only_associated column as an optional qual per https://docs.aws.amazon.com/inspector/v2/APIReference/API_ListMembers.html